src/overlays/actors/ovl_Obj_Jg_Gakki/z_obj_jg_gakki.c

```diff
@@ -28,14 +27,38 @@ const ActorInit Obj_Jg_Gakki_InitVars = {
     (ActorFunc)ObjJgGakki_Draw,
 };
 
-#endif
+extern AnimationHeader D_0601B1E8;
+extern SkeletonHeader D_0601B210;
 
-extern UNK_TYPE D_0601B1E8;
+void ObjJgGakki_Init(Actor* thisx, GlobalContext* globalCtx2) {
+    GlobalContext* globalCtx = globalCtx2;
+    ObjJgGakki* this = (ObjJgGakki*)thisx;
```

Comment:
Currently we use the `THIS` macro for this specific recast
```suggestion
    ObjJgGakki* this = THIS;
```